linux-glibc/build_container/getenvoy_linux_glibc_rbe_fips.sh

```diff
@@ -22,4 +22,4 @@ curl -sSL https://releases.llvm.org/9.0.0/clang+llvm-9.0.0-x86_64-linux-gnu-ubun
   tar Jx --strip-components=1 -C /usr/local
 
 # Force libc++ to be a static link by putting a linker script to do that.
-echo 'INPUT(-l:libc++.a -l:libc++abi.a -lm -lpthread)' > /usr/local/lib/libc++.so
+# echo 'INPUT(-l:libc++.a -l:libc++abi.a -lm -lpthread)' > /usr/local/lib/libc++.so
```

Comment:
This needs to be removed from the image since it forces a static link of the lic++ library.  For FIPS, we want this to be dynamic in order to remove the downstream dependencies, which on Ubuntu require glibc2.18.  By doing dynamic linking, we can use a version compiled for CentOS 7 instead.